### ISSUE TITLE
Google.Protobuf.dll "fails" Microsoft Platform Ready Test

### ISSUE THREAD

**Issue by fmuecke** [google/protobuf#2257]
The platform ready test tool produces the following issues:

```
Google.Protobuf.dll does not have a valid "Company name" property.
Google.Protobuf.dll does not have a valid "Product name" property.
```

We used version 3.0.0 to run the test, but 3.1.0 still produces the same result. 

It should be easy to add the missing properties to assembly information.

**Comment by haberman** [comment]
This is tagged C++, but I think this might actually be for C#?

How do we set this? Could you submit a PR? Our core team doesn't have any C# experts.

**Comment by anandolee** [comment]
The assemblies are remove in PR #1727. 
@jskeet, are we intended to remove them?

**Comment by jskeet** [comment]
We rely on project.json to pick up on attributes, basically. We end up with `AssemblyTitleAttribute`, `AssemblyDescriptionAttribute` etc - but apparently not the attributes the tool is expecting. (It's annoying that it doesn't tell us the exact attribute name it's looking for, but a "human readable" (less useful) one.)

Looking at a couple of other .NET Core projects from Microsoft, this looks entirely normal... I can't say I've ever used the "Platform Ready" tool. We could add attributes if we really need to though.

**Comment by anandolee** [comment]
@fmuecke can you submit a PR to add the attributes?

**Comment by fmuecke** [comment]
![image](https://cloud.githubusercontent.com/assets/7921024/23918792/ae92c932-08f4-11e7-9ab7-88196a3633a0.png)

Seems to be resolved with current 3.2.x.
